frontend/test/metabase/lib/expressions/suggest.unit.spec.js

```diff
@@ -224,9 +224,9 @@ describe("metabase/lib/expression/suggest", () => {
 
       it("should suggest partial matches in aggregation", () => {
         expect(suggest({ source: "1 + C", ...aggregationOpts })).toEqual([
-          { type: "aggregations", text: "Count " },
+          { type: "aggregations", text: "Count(" },
```

Comment:
See [this](https://metaboat.slack.com/archives/C03SHMZ266R/p1668683144141639?thread_ts=1668676917.056199&cid=C03SHMZ266R) slack thread for rationale to change autocomplete behaviour of `count` to `count()`

Reply:
Yes, I think it warrants a separate PR to facilitate isolated testing on that behavior change.
For instance, if it's not done properly, then **all** existing custom expressions that contains `count` (sans the parentheses) will appear as `count()` and it can be mildly (or not, depending on the users) _annoying_.

Reply:
Okay, I'll make a separate PR for this.

> all existing custom expressions that contains count (sans the parentheses) will appear as count() and it can be mildly (or not, depending on the users) annoying.

To clarify, that is actually the intended outcome of this change. Anyway, it's worth looking at in the coming PR.

Reply:
New PR is [here](https://github.com/metabase/metabase/pull/26645).